response_operations_ui/forms.py

```diff
@@ -274,8 +274,6 @@ def validate_day(form, field):
 class CreateCollectionExerciseDetailsForm(FlaskForm):
     user_description = StringField("user_description")
     period = IntegerField("period")
-    hidden_survey_id = HiddenField("hidden_survey_id")
```

Comment:
We should try to avoid doing this, passing around hidden fields which naturally can be changed on the UI isn't a good idea. No extra calls have been made as I have moved it in the backend